test/provisioner/common.go

```diff
@@ -0,0 +1,64 @@
+package provisioner
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+)
+
+type patchLabel struct {
+	Op    string            `json:"op"`
+	Path  string            `json:"path"`
+	Value map[string]string `json:"value"`
+}
+
+func AddNodeRoleWorkerLabel(clusterName string, cfg *envconf.Config) error {
+	fmt.Printf("Adding worker label to nodes belonging to: %s\n", clusterName)
+
+	config, err := clientcmd.BuildConfigFromFlags("", cfg.KubeconfigFile())
+	if err != nil {
+		return err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("initialized variables\n")
+	nodelist, err := clientset.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	label := map[string]string{"node-role.kubernetes.io/worker": ""}
```

Comment:
Depending on merge order, this might needs to updated to fit the changes in https://github.com/confidential-containers/cloud-api-adaptor/pull/784